### DISCUSSION TITLE
How can i disable the "s" keybind

### DISCUSSION THREAD

**Discussion by serfrae** [folke/trouble.nvim#553] [General]
I use flash.nvim for navigation rather heavily, I'd like to remove the mapping in trouble so I can use it's functionality, currently I'm having to navigate out of the window

**Comment by Heradar** [comment]
It looks like you can't remove default keybindings, only replace them. I did this but for leap.nvim

```
s = function() require("flash").jump() end
```

**Reply by folke** [reply]
Just pushed a change where you can now set `opts.keys.s = false`

**Reply by serfrae** [reply]
Amazing, cheers!
